.travis.yml

```diff
@@ -34,6 +34,8 @@ install:
 script:
 - flake8 pyccl
 - flake8 --exclude=data benchmarks
+- cd doc/0000-ccl_note && make && cd ..
+- cd doc/ccl_paper && make apj && cd ..
```

Comment:
I'm guessing this won't work because it's two layers deep.
I'd suggest
```
pushd doc/000-ccl_note && make && popd
pushd doc/ccl_paper && make && popd
```

Reply:
True, let's see if it passes now.